ui/models.py

```diff
@@ -20,6 +20,16 @@
 TRANSCODE_PREFIX = 'transcoded'
 
 
+class Channel(models.Model):
+    """
+    Model for Video Channels
+    """
+    title = models.TextField()
+    description = models.TextField(null=True, blank=True)
+    slug = models.UUIDField(unique=True, null=False, blank=False, default=uuid4)
+    owners = models.ManyToManyField(settings.AUTH_USER_MODEL)
```

Comment:
Not really. The idea was to avoid to corner ourselves in a spot where we need to have different users that actually own the channel.
But the more I think about it, the less I am convinced.
I think I will move it to a foreignKey